tests/org.codechecker.eclipse.rcp.it.tests/pom.xml

```diff
@@ -24,21 +24,9 @@
                     <useUIThread>false</useUIThread>
                     <product>org.codechecker.eclipse.feature</product>
                     <application>org.eclipse.ui.ide.workbench</application>
+                    <testClass>org.codechecker.eclipse.plugin.AllTests</testClass>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
This section was moved to the test root pom.

Reply:
Because the Shared project is already Checkstyle compliant. Now the default for all test projects is to fail on violations, but expilcitly specified unit tests to **not** to fail on violations until further refactor.